server/prisma/schema.prisma

```diff
@@ -15,11 +15,25 @@ model User {
   email String  @unique
   name  String?
   isPointOfContact Distributor[] @relation("isPointOfContact")
+  checkedInPallot Food_Entry[] @relation("checkedInPallot")
 
 }
 
 model Distributor {
   id Int @id @default(autoincrement())
   name String
   user User[] @relation("isPointOfContact")
+  isDistributor Food_Entry[] @relation("isDistributor")
+}
+
+model Food_Entry {
+  entererName     User[] @relation("checkedInPallot") // this shouldn't be User[], just User but what are the fields?
+  inputDate       DateTime @default(now())
+  expirationDate  DateTime?
```

Comment:
If we just want a date for expirationDate, add the @db.date tag after the DateTime?